overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -76185,7 +76185,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Heavy Metal",
-          "desc:8": "Craft a Metallurgic Infuser! This is the main machine you\u0027ll use to create all the various Mekanism components. Like for all Mekanism machines, you can simply connect an Immersive Engineering Wire Coil to it by placing a Wire Connector on any side. Once it\u0027s powered, place the main material in the second slot from the left, and the infusing material in the first slot from the left. Note that the machine can only hold and use one type of infusing material at once, meaning you\u0027re most likely better off creating multiple Metallurgic Infusers for maximum efficiency. To automate the machine, click on the Side Config button on the left. You can select which side you want items, infusion materials and energy to be input on, and which side you want to use as output. Make sure to push and pull items in/out with a Hopper or an Itemduct with an active Servo on it. And one more thing: All Mekanism machines can be picked up and moved by a Crescent Hammer from Thermal Expansion, so you don\u0027t need to worry about energy loss."
+          "desc:8": "Craft a Metallurgic Infuser! This is the main machine you\u0027ll use to create all the various Mekanism components. Like for all Mekanism machines, you can simply connect an Immersive Engineering Wire Coil to it by placing a Wire Connector on any side. Once it\u0027s powered, place the main material in the second slot from the left, and the infusing material in the first slot from the left. Note that the machine can only hold and use one type of infusing material at once, meaning you\u0027re most likely better off creating multiple Metallurgic Infusers for maximum efficiency. To automate the machine, click on the Side Config button on the left. You can select which side you want items, infusion materials and energy to be input on, and which side you want to use as output. Make sure to push and pull items in/out with a Hopper or an Itemduct with an active Servo on it. And one more thing: All Mekanism machines can be picked up and moved by a Crescent Hammer from Thermal Expansion, so you don\u0027t need to worry about energy loss.\n\nNote: As soon as you have an Enrichment Chamber, make sure to use Compressed Carbon/Redstone/Diamonds for enrichment to save resources. Every compressed resource allows for the infusion of eight items."
```

Comment:
please avoid telling people what to do in this manner. Instead, keep it informational, such as `Once you have the Enrichment Chamber, you can convert Coal, Redstone, or Diamonds into Compressed Carbon, Redstone, or Diamond to multiply the amount of infusion gained per material by 8.`
additionally, perhaps this information would be better suited to the Enrichment Chamber quest?

Reply:
I was debating whether I should put it in both quests but that seemed a little redundant. I put it with the Metallurgic Infuser because that is where you get the advantage of the compressed items. I also had a reason based on the layout of the two quests but I'll have to look at that in detail when I open the game again tonight or tomorrow.